src/components/App/App.css

```diff
@@ -0,0 +1,20 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: rgba(30, 6, 9, 1);
+}
```

Comment:
Can we change this to   `background-color: rgba(29,0,7);` that way it matches the border of the banner and you can't see the edges